src/Guesser/FilterTypeGuesser.php

```diff
@@ -73,36 +82,46 @@ public function guessType($class, $property, ModelManagerInterface $modelManager
         $options['field_name'] = $metadata->fieldMappings[$propertyName]['fieldName'];
 
         switch ($metadata->getTypeOfField($propertyName)) {
-            case 'boolean':
+            case Type::BOOL:
+            case Type::BOOLEAN:
                 $options['field_type'] = BooleanType::class;
                 $options['field_options'] = [];
 
-                return new TypeGuess('doctrine_mongo_boolean', $options, Guess::HIGH_CONFIDENCE);
-//            case 'datetime':
-//            case 'vardatetime':
```

Comment:
Why did you remove this? This is a BC break

Reply:
If you mean 
```php
return new TypeGuess('doctrine_mongo_boolean', $options, Guess::HIGH_CONFIDENCE);
```
has been replaced by 
```php
return new TypeGuess(BooleanFilter::class, $options, Guess::HIGH_CONFIDENCE);
```
The rest of this code was commented out and I've uncommented it using the constant types where the type existed.

Reply:
Oh, you're right :smile: